services/api/src/resources/project/resolvers.ts

```diff
@@ -611,6 +621,12 @@ export const updateProject: ResolverFn = async (
     }
   }
 
+  if(typeof sharedBaasBucket == "boolean") {
+    if (adminScopes.projectViewAll) {
+      throw new Error('Setting shared baas bucket is only available to administrators.');
+    }
+  }
+
```

Comment:
The PR description and this authz check suggest that updating `sharedBaasBucket` should be possible, but there are still some missing pieces to complete that: The update/patch types for project need to be updated in `typeDefs.js`; The `sql.updateProject` call needs to get passed the input later on in this function, and the argument needs to get added to the function definition in `project/sql.ts`.

Reply:
Ah thanks